packages/musap-rn-kms/src/agent/MusapKeyManagerSystem.ts

```diff
@@ -0,0 +1,127 @@
+import { IKey, ManagedKeyInfo, MinimalImportableKey, TKeyType } from '@veramo/core';
+import {
+  KeyAlgorithmType,
+  KeyGenReq,
+  MusapKey,
+  MusapModuleType, signatureAlgorithmFromKeyAlgorithm,
+  SignatureAlgorithmType,
+  SignatureFormat,
+  SignatureReq,
+} from '@sphereon/musap-react-native'
+import { SscdType } from '@sphereon/musap-react-native/src/types/musap-types';
+import { KeyManagementSystem } from '@veramo/kms-local';
+import { AbstractPrivateKeyStore } from '@veramo/key-manager';
+import 'react-native-get-random-values'
+import { v4 as uuid } from 'uuid';
+import { TextDecoder } from 'text-encoding';
+import Debug from 'debug'
+
+const debug = Debug('sphereon:musap-rn-kms')
+
+export class MusapKeyManagementSystem extends KeyManagementSystem {
```

Comment:
This KMS really should not have extended a KMS that is about local keys.

Reply:
switched to use `AbstractKeyManagementSystem`